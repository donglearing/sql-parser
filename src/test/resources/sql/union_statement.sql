select a from T union select b from F;