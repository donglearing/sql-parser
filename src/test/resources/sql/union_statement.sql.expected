SELECT a FROM T UNION SELECT b FROM F